docs/docsite/rst/collections_guide/collections_installing.rst

```diff
@@ -185,7 +185,7 @@ Using locally installed collections adjacent to playbooks has some benefits, suc
 
 Here is an example of keeping a collection adjacent to the current playbook, under a ``collections/ansible_collections/`` directory structure.
 
-.. code-block:: text
+.. code-block:: console
```

Comment:
It's not really `console`. Console is for snippets copied from terminal output, featuring a prompt followed by its command and then the output.